components/event-service/internal/externalapi/genericeventhandler_test.go

```diff
@@ -20,101 +20,116 @@ func TestNewEventsHandler(t *testing.T) {
 		requestSize = 65536
 		v1Endpoint  = "/" + source + "/v1/events"
 	)
-	meshURL, closeFn := meshtesting.MockEventMesh(t)
-	defer closeFn()
-
-	conf, err := mesh.InitConfig(source, meshURL)
-	if err != nil {
-		t.Fatalf("Error init config: %s", err)
-	}
-
-	mux := http.NewServeMux()
-	mux.Handle(v1Endpoint, NewEventsHandler(conf, requestSize))
 
 	tests := []struct {
-		name           string
-		givenPayload   string
-		wantCode       int
-		wantEventID    string
-		wantErrorType  string
-		wantErrorField string
+		name                      string
+		givenPayload              string
+		withEventMeshResponseCode int
```

Comment:
(nit) I think the original authors wanted to follow [this principe](https://github.com/uber-go/guide/blob/master/style.md#test-tables) there all input data for the test is prefixed with `given`, and the expected results are prefixed with `want`.

```suggestion
		givenEventMeshResponseCode int
```

Reply:
Same comment!

Reply:
Fixed